engine/objconv/mesher/to_OgreMesh.cpp

```diff
@@ -440,7 +440,7 @@ static void SetupColorTplVars( map< string, string > &vars,
                                bool force_rgba = false )
 {
     bool   eqr   = (fabs( defr-r ) < 0.001);
-    bool   eqg   = (fabs( defg-g ) < 0.001);
+    //bool   eqg   = (fabs( defg-g ) < 0.001);
```

Comment:
The real bug here seems to be the lack of including `eqg` on line 446 as this effectively removes green from the RGBA calculation. Either that or a comment needs to be provided as to why it's being dropped.
@LifWirser  thoughts? Should we take it commented out? Or include it in 446 with a note about history in case a bug is found?

Reply:
I added it in 446 with a note